CMakeLists.txt

```diff
@@ -80,6 +80,9 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CHEMFILES_SANITIZERS}")
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${CHEMFILES_SANITIZERS}")
 set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} ${CHEMFILES_SANITIZERS}")
 
+# Omit literal operator `_sv` in external string-view-lite
+add_definitions("-Dnssv_CONFIG_USR_SV_OPERATOR=0")
```

Comment:
Specifying a single target does not work because the header is used by all the test targets of text files as well.
Instead, I just edited the default setting in the header directly.